docs/tutorials/multinode/noise.py

```diff
@@ -0,0 +1,51 @@
+import cirq
```

Comment:
Consider adding a docstring at the beginning to give an overview of this file.  Same for other python files.